Defs/Ammo/HighCaliber/14.5x114mmSoviet.xml

```diff
@@ -164,30 +187,65 @@
     <products>
       <Ammo_145x114mm_FMJ>200</Ammo_145x114mm_FMJ>
     </products>
-    <workAmount>7000</workAmount>
+    <workAmount>7600</workAmount>
+  </RecipeDef>
+
+  <RecipeDef ParentName="AmmoRecipeBase">
+    <defName>MakeAmmo_145x114mm_Incendiary</defName>
+    <label>make 14.5x114mm (AP-I) cartridge x200</label>
+    <description>Craft 200 14.5x114mm (AP-I) cartridges.</description>
+    <jobString>Making 14.5x114mm (AP-I) cartridges.</jobString>
+    <ingredients>
+      <li>
+        <filter>
+          <thingDefs>
+            <li>Steel</li>
+          </thingDefs>
+        </filter>
+        <count>76</count>
+      </li>
+      <li>
+        <filter>
+          <thingDefs>
+            <li>Prometheum</li>
+          </thingDefs>
+        </filter>
+        <count>7</count>
+      </li>
+    </ingredients>
+    <fixedIngredientFilter>
+      <thingDefs>
+        <li>Steel</li>
+        <li>Prometheum</li>
+      </thingDefs>
+    </fixedIngredientFilter>
+    <products>
+      <Ammo_145x114mm_Incendiary>200</Ammo_145x114mm_Incendiary>
+    </products>
+    <workAmount>10400</workAmount>
   </RecipeDef>
 
   <RecipeDef ParentName="AmmoRecipeBase">
     <defName>MakeAmmo_145x114mm_HE</defName>
-    <label>make 14.5x114mm (HE) cartridge x200</label>
-    <description>Craft 200 14.5x114mm (HE) cartridges.</description>
-    <jobString>Making 14.5x114mm (HE) cartridges.</jobString>
+    <label>make 14.5x114mm (AP-HE) cartridge x200</label>
```

Comment:
This should just be HE

Reply:
Fixed. Was going by Stat Sheet ammo type for labels.